src/ContainerIntegration.jsx

```diff
@@ -92,7 +92,7 @@ const ContainerEnv = ({ containerEnv, imageEnv }) => {
 const ContainerIntegration = ({ container, containerDetail, localImages }) => {
     if (containerDetail === null || localImages === null) {
         return (
-            <EmptyStatePanel title={_('Loading details...')} loading />
+            <EmptyStatePanel title={_("Loading details...")} loading />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1312-20230622-151208-7385b7d9-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)